### ISSUE TITLE
Add option to disable swiftlint in-file

### ISSUE THREAD

**Issue by fredpi** [realm/SwiftLint#2071]
For some cases, it would be useful to have a command to disable swiftlint parsing for a specific file (e. g. `// swiftlint:disable:file`.

I know there's already an exclude option in the .yml file, but that one is not always suitable. Consider 3rd-party-frameworks like [SwiftGen](https://github.com/SwiftGen/SwiftGen) that currently always have to specifically disable single rules. A disable command in-file would fix that issue.

**Comment by marcelofabri** [comment]
We probably should follow [rubocop's approach](http://rubocop.readthedocs.io/en/latest/configuration/):

```swift
// swiftlint:disable all
[...]
// swiftlint:enable all
```

@fredpi Would you like to submit a PR implementing this feature?

**Comment by fredpi** [comment]
Yeah, will do a PR with an implementation these days

**Comment by fredpi** [comment]
Didn't forget about this, however, will have to delay this a few days. On a tight schedule currently...

**Comment by marcelofabri** [comment]
No worries!

**Comment by bj97301** [comment]
Hows it going?

**Comment by fredpi** [comment]
@bj97301 As pointed out, takes a bit, but looking good...

**Comment by bj97301** [comment]
Nice.  Looking forward to it :D

**Comment by marcelofabri** [comment]
Closed in #2095 by @fredpi 💯
